src/app/components/users-panel/users-panel.component.html

```diff
@@ -15,9 +15,55 @@ <h4> Current Members </h4>
     <div class="">
       <h4> Map </h4>
       <h6>Name:</h6>
-      <p>{{activeProject.name}}</p>
+      <div id=project-container>
+        <div id=subproject-container >
+          <div id=project-name
+               data-tooltip
+               tabindex="1"
+               title="{{activeProject.name}}"
+               data-position="top"
+               data-alignment="left"
+               *ngIf="!nameInputShown"
+               (dblclick)=showNameInput()>
+            {{activeProject.name}}
+          </div>
+          <input #activeProjectName
+                 id=project-name-input
+                 type="text"
+                 *ngIf="nameInputShown"
+                 (blur)="onEnterName(activeProjectName.value)"
+                 (keyup.enter)="onEnterName(activeProjectName.value)">
+        </div>
+        <i class="fas fa-edit" style="margin: auto" (click)="showNameInput()" *ngIf="!nameInputShown"></i>
+        <i class="fas fa-check" style="margin: auto" (click)="hideNameInput()" *ngIf="nameInputShown"></i>
+      </div>
+      <div class="display-error" *ngIf="nameInputError">
+        {{nameErrorMessage}}
+      </div>
+
       <h6>Description:</h6>
-      <p>{{activeProject.description}}</p>
+      <div style="display: flex; justify-content: space-between">
+        <div style="display: inline-block; width: 100%">
```

Comment:
I'm also curious if a new component should be created to wrap up the edit input.  Not certain if that is overkill but used 2 places here and could be used other places going forward.

Reply:
@nathanfranklin I'll make a ticket for that.

Reply:
> @nathanfranklin I'll make a ticket for that.

perfect!